react-js/src/pages/viewDocuments-styles.scss

```diff
@@ -0,0 +1,108 @@
+@import '../library';
+
+.component-cropping {
+    height: 100%;
+}
+
+.component-imageResults,
+.component-imageDetail,
+.component-cropping {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    min-height: 100%;
+    background-color: $color-background-lightGrey;
+
+    .resultsGridContainer {
+        max-width: 1140px;
+        width: 100%;
+        margin: 0 auto;
+        flex: 1;
+
+        // .resultsGrid {
```

Comment:
at my previous job we used to call removing deprecated comments "plucking chickens"